spec/factories/proceeding_types.rb

```diff
@@ -16,6 +16,8 @@
     default_cost_limitation_substantive { 2 }
     involvement_type_applicant { false }
     additional_search_terms { nil }
+    start_date { '2018-08-07' }
+    end_date { '2021-09-12' }
```

Comment:
I'm wondering if this is going to cause problems with the tests when we run them after 12/09?

Reply:
If I set this to use e.g Time.zone.now +/- 2.years, it causes a test to fail as the new proceeding type that is created is different to the one in the DB, so when we expect it to not create a duplicate row it actually does.

Reply:
If I set this to use e.g Time.zone.now +/- 2.years, it causes a test to fail as the new proceeding type that is created is different to the one in the DB, so when we expect it to not create a duplicate row it actually does. I will do some more checks to see how it affects things

Reply:
But if I backdate the `end_date` to `2021-09-09` then `spec/requests/proceeding_types/index_proceeding_types_spec.rb` fails. So as soon. as we pass 12/09 we're going to start seeing failures. 

If we can't change it in the factory then maybe we need to override it in that spec, change the setup to something like 
 ```     
create(:proceeding_type,
               :with_real_data,
               code: 'PH0002',
               ccms_code: 'PBM24',
               ccms_category_law: 'Other',
               ccms_matter: 'Public Law - Other',
               ccms_matter_code: 'APBLB',
               meaning: 'Not an application for a care order',
               description: 'Not to be represented on an application for a care order.',
               additional_search_terms: 'injunction',
               end_date: Time.zone.now + 2.years)

Reply:
thanks